src/components/web/home/home-screen-web.scss

```diff
@@ -49,34 +50,37 @@
     }
   }
 
-  .play {
+  .button {
```

Comment:
revert this to 'play'! this is your custom classname for the button, so should style against that